enterprise/api/v1/views/enterprise_group.py

```diff
@@ -112,16 +113,19 @@ def get_learners(self, *args, **kwargs):
                         'learner_uuid': 'enterprise_customer_user_id',
                         'pending_learner_id': 'pending_enterprise_customer_user_id',
                         'enterprise_group_membership_uuid': 'enterprise_group_membership_uuid',
+                        'enterprise_customer': EnterpriseCustomerSerializer
                     },
                 ],
             }
 
         """
 
         group_uuid = kwargs.get('group_uuid')
+        # GET api/v1/enterprise-group/<group_uuid>/learners?filter_for_pecus=true
+        filter_for_pecus = self.request.query_params.get('filter_for_pecus', None)
```

Comment:
Also, it'd be good to add something to the docstring for this method about `filter_for_pecus` so that it's knowable from our API docs.

Reply:
Naming nit for this query param: maybe `pending_users_only` to avoid any ambiguity around the word "filter"?